<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchDialog</class>
 <widget class="QDialog" name="BatchDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch job</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelSettingsPath">
     <property name="text">
      <string>Setttings file:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditSettingsFile"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonBrowseSettings">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelSize">
     <property name="text">
      <string>Size of lattice:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboSize">
     <item>
      <property name="text">
       <string>64 x 64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128 x 128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256 x 256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512 x 512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1024 x 1024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128 x 64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256 x 128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512 x 256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1024 x 512</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelSimulations">
     <property name="text">
      <string>Number of simulations:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBoxSimulations">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelSeasons">
     <property name="text">
      <string>Number of seasons:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBoxSeasons">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelFilenamePrefix">
     <property name="text">
      <string>Filename prefix:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditFilenamePrefix">
     <property name="text">
      <string>results</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelPath">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEditPath"/>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="buttonBrowse">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="8">
    <widget class="QListWidget" name="listWidgetJobs"/>
   </item>
  </layout>
  <zorder>labelSize</zorder>
  <zorder>comboSize</zorder>
  <zorder>labelSimulations</zorder>
  <zorder>spinBoxSimulations</zorder>
  <zorder>labelSeasons</zorder>
  <zorder>spinBoxSeasons</zorder>
  <zorder>labelFilenamePrefix</zorder>
  <zorder>lineEditFilenamePrefix</zorder>
  <zorder>labelPath</zorder>
  <zorder>lineEditPath</zorder>
  <zorder>buttonBrowse</zorder>
  <zorder>listWidgetJobs</zorder>
  <zorder>labelSettingsPath</zorder>
  <zorder>lineEditSettingsFile</zorder>
  <zorder>pushButtonBrowseSettings</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>BatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
